### PR TITLE
[Feature] Setup database

### PR DESCRIPTION
**Pull request by johnkahn**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../THIRD-PARTY-NOTICES.txt)

## Description of Changes
Created the database entities and migrations

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'reflect-metadata';
```
Comment (SpencerKaiser) [inline src/index.ts]: What's this do?

File: .eslintrc.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ module.exports = {
     'import/no-extraneous-dependencies': ['error', { devDependencies: ['**/*.test.ts'] }],
     'max-len': [0],
     'import/prefer-default-export': ['off'],
+    'object-curly-newline': ['off'],
```
Comment (SpencerKaiser) [inline .eslintrc.js]: Doesn't this open the door to some funky declarations?

File: src/index.ts (hunk 1 above)
Comment (johnkahn) [reply src/index.ts]: It's for the decorators in the entities. The ✨ magic ✨ behind TypeORM for those beautiful db classes

Comment (johnkahn) [reply src/index.ts]: No clue how it works tbh

File: .eslintrc.js (hunk 2 above)
Comment (johnkahn) [reply .eslintrc.js]: The issue was it's erroring on this
```
import { Entity, PrimaryGeneratedColumn, Column, BaseEntity } from 'typeorm';
```
it wants
```
import {
   Entity,
   PrimaryGeneratedColumn,
   Column,
   BaseEntity
} from 'typeorm';
```

Comment (johnkahn) [reply .eslintrc.js]: Oh wait, it looks like it's just a difference of `prettier` vs `eslint`. Yeah I'll fix that

Comment (johnkahn) [reply .eslintrc.js]: Ok, so the issue is that `prettier` will break it into multiline based on line length and `eslint` will break it based on the number of properties. So disabling it for `eslint` means `prettier` still catches it. IMO I prefer `prettier`'s method.

**Pull request merged**